src/components/css/BookStore.module.css

```diff
@@ -1,6 +1,8 @@
-.BooksContainer {
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  gap: 1em;
-}
+.BooksContainer {
+  /* width: 100%; */

```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. Kindly remove all unused code this will make your code much noise, It is always a good practice to keep your code clean from such comments. 😉 👍🏻